docs/source/faq.rst

```diff
@@ -142,12 +142,11 @@ Please refer to :class:`optuna.logging` for further details.
 How to save machine learning models trained in objective functions?
 -------------------------------------------------------------------
 
-Optuna saves hyperparameter values with its corresponding objective value to storage,
+Optuna saves hyperparameter values with their corresponding objective values to storage, 
 but it discards intermediate objects such as machine learning models and neural network weights.
-To save models or weights, please use features of the machine learning library you used.
 
-We recommend saving :obj:`optuna.trial.Trial.number` with a model in order to identify its corresponding trial.
-For example, you can save SVM models trained in the objective function as follows:
+To save models or weights, we recommend utilizing Optuna's built-in ``ArtifactStore``.
+For example, you can use the :func:`~optuna.artifacts.upload_artifact` as follows:
 
 .. code-block:: python
 
```

Comment:
It would be better to add the instruction of the artifact store setting as follows.
```suggestion
base_path = "./artifacts"
os.makedirs(base_path, exist_ok=True)
artifact_store = optuna.artifacts.FileSystemArtifactStore(base_path=base_path)
```